drivers/storage/class/disk_new/data.c

```diff
@@ -64,11 +68,17 @@ CLASSPNP_SCAN_FOR_SPECIAL_INFO DiskBadControllers[] = {
 // Added MediaTypes in their own brace nesting level
 // ======== ROS DIFF ========
 
+#ifdef __REACTOS__
+DATA_SEG("PAGERDAT")
```

Comment:
The standard name (see e.g. WDK' makefile.new; section names in windows' ntoskrnl, etc...) seem to be "PAGECONS" (or "PAGECONST", but the 'T' gets trimmed away)

Reply:
PAGECONS it is, then